packages/client/hmi-client/src/types/Provenance.ts

```diff
@@ -0,0 +1,37 @@
+export enum ProvenanceType {
+	// eslint-disable-next-line @typescript-eslint/no-shadow
+	Dataset = 'Dataset',
+	// eslint-disable-next-line @typescript-eslint/no-shadow
+	Model = 'Model',
```

Comment:
Why do you need those `eslint-disable-...` if `Dataset` and `Model` aren't defined outside the export scope?

Reply:
that was needed earlier but not anymore so I have removed it